### ISSUE TITLE
Adding a new form group fails if `allowAdminChanges` is  set to`false`, even though no change will be written to project config

### ISSUE THREAD

**Issue by tomdavies** [barrelstrength/craft-sprout-forms#371]
### Description 

Adding a new form group in Sprout Forms fails if administrative changes are disabled for the current environment, even though form groups are not written to `config/project.yaml`

### Expected behaviour

_Either_ that I can create form groups when `'allowAdminChanges' => false` is set, _or_ that form groups are tracked in project config (ideally the former).

### Steps to reproduce

1. Set `'allowAdminChanges' => false` in `config/general.php`
2.Navigate to `sprout-forms/forms` in CP UI
3. Click "New Group" and enter a group name and click "ok"
4. Everyone's favourite "An unknown error occurred" is displayed in the flash

Inspecting `web.log` yields: 

```
2019-11-20 10:45:09 [-][1][-][trace][craft\web\Application::_processActionRequest] Route requested: 'sprout-forms/groups/save-group'
2019-11-20 10:45:09 [-][1][-][trace][yii\base\Controller::runAction] Route to run: sprout-forms/groups/save-group
2019-11-20 10:45:09 [-][1][-][trace][yii\base\InlineAction::runWithParams] Running action: barrelstrength\sproutforms\controllers\GroupsController::actionSaveGroup()
2019-11-20 10:45:09 [-][1][-][error][yii\web\HttpException:403] yii\web\ForbiddenHttpException: Administrative changes are disallowed in this environment. in /app/vendor/craftcms/cms/src/web/Controller.php:283
Stack trace:
#0 /app/vendor/barrelstrength/sprout-forms/src/controllers/GroupsController.php(27): craft\web\Controller->requireAdmin()
#1 [internal function]: barrelstrength\sproutforms\controllers\GroupsController->actionSaveGroup()
#2 /app/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#3 /app/vendor/yiisoft/yii2/base/Controller.php(157): yii\base\InlineAction->runWithParams(Array)
#4 /app/vendor/craftcms/cms/src/web/Controller.php(187): yii\base\Controller->runAction('save-group', Array)
#5 /app/vendor/yiisoft/yii2/base/Module.php(528): craft\web\Controller->runAction('save-group', Array)
#6 /app/vendor/craftcms/cms/src/web/Application.php(299): yii\base\Module->runAction('sprout-forms/gr...', Array)
#7 /app/vendor/craftcms/cms/src/web/Application.php(565): craft\web\Application->runAction('sprout-forms/gr...', Array)
#8 /app/vendor/craftcms/cms/src/web/Application.php(278): craft\web\Application->_processActionRequest(Object(craft\web\Request))
#9 /app/vendor/yiisoft/yii2/base/Application.php(386): craft\web\Application->handleRequest(Object(craft\web\Request))
#10 /app/web/index.php(21): yii\base\Application->run()
```

### Additional info

- Craft version: 3.3.15
- PHP version: 7.3.7
- Database driver & version:  MySQL 5.7.18
- Plugins & versions: Sprout Forms 3.6.1

**Comment by BenParizek** [comment]
Thanks @tomdavies.

We'll review and get these instances cleaned up.

**Comment by BenParizek** [comment]
@tomdavies This should be resolved in Sprout Forms v3.6.4
